app/_locales/zh_CN/messages.json

```
@@ -413,19 +413,19 @@
     "message": "功能类型"
   },
   "gasLimitInfoTooltipContent": {
-    "message": "天然气限制即您个人希望购买的最大单位数量。"
+    "message": "gasLimit 即您个人希望付出的最大 Gas 数量。"
```

There may be a more appropriate word than 限制, but we should definitely use the appropriate Mandarin for "limit" here:

```suggestion
Gas 的限制即您个人希望付出的最大 Gas 数量
```

In addition, we should add a message for `gasLimit`, which ought to be "Gas 的限制" or whatever is appropriate.

---

I believe the following best captures the translation of: "Gas limit is the maximum amount of units of gas you are willing to spend."

```suggestion
    "message": "gasLimit 是您个人希望付出的最大 Gas 数量。"
```

---

越简单越好。

---

Per [this website](https://cypherpunks-core.github.io/ethereumbook_zh/13.html), I believe we should make the following changes @wanshan1024:
```
  "gasLimit": {
    "message": "Gas 限制"
  },
  "gasLimitInfoTooltipContent": {
    "message": "Gas 限制是您个人希望付出的最大 Gas 数量。"
  },
  "gasLimitTooLow": {
    "message": "Gas 限制至少要 21000"
  },
```

---

Please let me know if you disagree!

---

agree, thanks, I will change it later.